tests/transforms/test_condition.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+from pennylane import numpy as np
+
+import pennylane as qml
+from pennylane.transforms.condition import ConditionalTransformError
+
+
+class TestCond:
+    """Tests that verify that the cond transform works as expect."""
```

Comment:
Minor: the name of this test class, the docstring, and the fact that it is the only test class, imply that it is not needed. You could remove it and just have test functions in this file.

Reply:
There is now a `class TestOtherTransforms` to the file too :smile: What would be the benefit of not having a class btw?

Reply:
> What would be the benefit of not having a class btw?

Oh it's just that test classes don't really do anything, their main use is simply categorization of multiple tests within a module. 

I find that if there isn't any categorization needed, it's cleaner to just have test functions.